### DISCUSSION TITLE
Publishing of mattermost4j 0.22.2?

### DISCUSSION THREAD

**Discussion by cgrimm01** [maruTA-bis5/mattermost4j#434] [Q&A] [answered]
Any plans of publishing 0.22.2 into a common maven repository?  Or maybe is there one that I should be pointing to as an alternative?

**Comment by maruTA-bis5** [comment, marked as answer]
I don't have any planned date of release 0.22.2 but I wish to do that this month (December 2021) after merging some PRs.
If you have any feature requests or bug reports, feel free to open a new issue.
